src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -15,7 +15,7 @@
 
       The supported mechanism for providing new compilers in a build enviroment is updating to the newer .NET SDK or Visual Studio Build Tools SKU.
 
-      This package requires either MSBuild 16.3 and .NET Desktop 4.7.2+ or .NET Core 2.1+
+      This package requires either MSBuild 16.3 and .NET Framework 4.7.2+ or .NET Core 3.1.
```

Comment:
```suggestion
      This package requires either MSBuild 16.3 and .NET Framework 4.7.2+ or .NET Core 3.1+.
```

Reply:
@jaredpar I am slightly surprised that csc, which is a .Net Core 3.1 app, would work on .Net 5, but it seems it does, so I have applied your change.

Reply:
IME most applications can safely roll forward to most major versions. 

In the case of the compiler though this is always true due to the nature of how the compiler ships. Consider that the same compiler (exact same binaries) are presently shipping in the .NET 3.1 and .NET 5 SDKs. Hence in order for this to work it must support execution on both `netcoreapp3.1` and `net50`. 

This is not specific to .NET SDK 3.1 / 5.0 but rather general to the current .NET SDK RTM and the one presently under development. Been this way since `netcoreapp2.0`.